### ISSUE TITLE
Fails on stored proceedures

### ISSUE THREAD

**Issue by spyhunter99** [BenoitDuffez/ScriptRunner#6]
This fails for me with postgres stored procedures

**Comment by BenoitDuffez** [comment]
I'm sorry to hear that. Feel free to fix the errors, I don't use postgres personnally.

**Comment by spyhunter99** [comment]
I think the root cause is that a stored procedure can have a semicolon on
it, yet it's technical one sql statement. There's a few keyword checks can
be added to detect this situation and treat it as a single command

**Comment by BenoitDuffez** [comment]
Did you overcome that issue?

**Comment by spyhunter99** [comment]
no, i worked around the issue by redesigning my database setup to no longer need the stored procedures
